packages/mwp-tracking-plugin/package.json

```diff
@@ -25,7 +25,7 @@
   "homepage": "https://github.com/meetup/meetup-web-platform#readme",
   "dependencies": {
     "@google-cloud/pubsub": "0.14.2",
-    "avsc": "5.0.5",
+    "avsc": "5.1.1",
```

Comment:
this package upgrade didn't make a practical difference, but it's good to stay up to date.